Maestro/subscriptions.json

```diff
@@ -317,6 +378,24 @@
         }
       }
     },
+    // Update dependencies in core-setup release/2.0.0
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/corefx/release/2.0.0/Latest.txt",
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/coreclr/release/2.0.0/Latest.txt",
```

Comment:
See `// Update dependencies in core-setup master` above

---
In reply to: [112553250](https://github.com/dotnet/versions/pull/145#discussion_r112553250) [](ancestors = 112553250)